docs/charts/match_weights_chart.ipynb

```diff
@@ -185,7 +213,60 @@
    "metadata": {},
    "source": [
     "### Actions to take as a result of the chart\n",
-    "\n"
+    "\n",
+    "Some heuristics to help assess Splink models with the `match_weights_chart`:\n",
+    "\n",
+    "#### Reducing match weights within a comparison\n",
```

Comment:
Maybe reword this subtitle? It makes it sound as if users should reduce the number of comparison levels in their comparison.

Contracting comparison match weights
or even just
Reducing comparison match weights

Reply:
have added alternate subtitle